tests/unit/Rules/AlwaysValidTest.php

```diff
@@ -17,11 +17,20 @@
 
 /**
  * @group  rule
+ *
  * @covers \Respect\Validation\Rules\AlwaysValid
+ *
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Paulo Eduardo <pauloelr@gmail.com>
+ * @author William Espindola <oi@williamespindola.com.br>
  */
-class AlwaysValidTest extends TestCase
+final class AlwaysValidTest extends TestCase
```

Comment:
Could you inherit `RuleTestCase` to keep consistent with the other tests?

Reply:
@batusa this rule always return valid and is not possible create a teste for invalid providers. To implement RuleTestCase we need to [implement valid and invalid providers](https://github.com/Respect/Validation/blob/master/tests/library/RuleTestCase.php#L41-L53), and in this case is not possible.